webroot/src/pages/FinalizePrivilegePurchase/FinalizePrivilegePurchase.vue

```diff
@@ -0,0 +1,171 @@
+                                        </div>
+                                        <InputNumber
+                                            :formInput="formData.expYear"
+                                            @input="handleExpYearInput(formData.expYear)"
+                                            @emitInputRef="handleExpYearRefEmitted"
+                                        />
+                                    </div>
+                                </div>
+                                <div class="cvv-container">
+                                    <InputNumber
+                                        :formInput="formData.cvv"
+                                        @input="handleCVVInput(formData.cvv)"
+                                        @emitInputRef="handleCVVRefEmitted"
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                        <div class="billing-address-section">
+                            <div class="billing-address-title">
+                                {{billingAddressTitleText}}
+                            </div>
+                            <div class="form-row">
+                                <InputText
+                                    :formInput="formData.streetAddress1"
+                                />
+                            </div>
+                            <div class="form-row">
+                                <InputText
+                                    :formInput="formData.streetAddress2"
+                                />
+                            </div>
+                            <div class="state-zip-line form-row">
+                                <InputSelect
+                                    :formInput="formData.stateSelect"
+                                />
+                                <InputNumber
+                                    :formInput="formData.zip"
+                                    @input="handleZipInput(formData.cvv)"
+                                />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="cost-breakdown-container">
+                    <div class="cost-listing-block">
+                        <div class="selection-title">{{selectionText}}</div>
+                        <ul>
+                            <li
+                                v-for="(state) in selectedStatePurchaseDataList"
+                                :key="state.jurisdiction.abbrev"
+                                class="selected-state-block"
+                            >
+                                <div class="info-row">
+                                    <div class="info-row-label">
+                                        {{`${state?.jurisdiction?.name()} ${compactPrivilegeStateFeeText}`}}
```

Comment:
I think from where it is now it might be easiest to do as a formatting method taking the state as an input.

Ill do the computed as its not too much harder but Im curious about your opinion on that approach in the general sense

Reply:
I'd probably be more tempted to handle it in the computed. I'd look to just add a `.map()` to the existing computed; or possibly create an additional computed if that's easier.